pkg/spdk/engine.go

```diff
@@ -205,7 +205,7 @@ func svcEngineGet(spdkClient *spdkclient.Client, name string) (res *spdkrpc.Engi
 		if !exists {
 			// This replica must be a local lvol
 			replicaName := spdktypes.GetLvolNameFromAlias(baseBdev.Name)
-			res.ReplicaAddressMap[replicaName] = ""
+			res.ReplicaAddressMap[replicaName] = fmt.Sprintf("%s:0", podIP)
```

Comment:
NIT: use `net.JoinHostPort()`

Reply:
One question:
Does local replica still allocate a port? @shuo-wu

Reply:
Updated. Thank you.